### ISSUE TITLE
A11y_AzureDataStudio_NewQuery_Keyboard: "Select Database" dropdown is not accessible using keyboard keys

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/azuredatastudio#22674]
"[Check out Accessibility Insights!] - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### #GitHub Tags:
#A11yMAS; #A11ySev2; #A11yTCS; #DesktopApp; #SH-AzureDataStudio-Linux-Jul2021; #Azure Data Studio; #WCAG2.1.1; #Linux; #Keyboard; 

### Environment Details:​
Application Name: MySQL extension for Azure Data Studio
Application Version: 1.42.0-insider
Operating System: Ubuntu 20.04.2 LTS
GNOME Version: 3.36.8
ORCA: 42.2-1ubuntu1
Distributor ID: Ubuntu
Description: Ubuntu 22.10
Release: 22.10
Codename: kinetic

### Repro Steps:

1. Launch Azure Data Studio application.
2. TAB to "Servers" and select any server from the list and hit ENTER on 'Connect' button.
3. TAB to "File" and hit ENTER key. TAB to "New Query" and hit ENTER key.
4. TAB to "Select database" and observe that "Select Database" dropdown is not accessible using keyboard keys.

### Actual Result:​
"Select Database" dropdown is not accessible using keyboard keys.

### Expected Result:​
"Select Database" dropdown should be accessible using keyboard keys.

### User Impact:
Keyboard users will not be able to get the information about these controls.

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**

### Attachments:

https://user-images.githubusercontent.com/90901519/228825393-17da1f12-ae5c-473e-a478-e228a4ed29c7.mp4

**Comment by PoojaNamde** [comment]
Primary Bug: https://github.com/microsoft/azuredatastudio-mysql/issues/179

**Comment by alanrenmsft** [comment]
@PoojaNamde in order to navigate out of the code editor you need to:
1. move the focus in a code editor
2. run the command `Toggle Tab Key Moves Focus`
![image](https://user-images.githubusercontent.com/13777222/236738718-bd862b78-c572-463b-9913-e458af8b46db.png)

and then you should be able to move the keyboard focus to the toolbar.
